src/ZipStream.php

```diff
@@ -21,12 +20,16 @@
  * @author Jesse Donat <donatj@gmail.com>
  * @copyright Copyright (C) 2014 Jesse Donat <donatj@gmail.com>
  *
- * @license https://raw.githubusercontent.com/maennchen/ZipStream-PHP/master/LICENCE
+ * @author Nik Barham <github@brokencube.co.uk>
+ * @copyright Copyright (C) 2016 Nik Barham <github@brokencube.co.uk>
+ *
+ * @license https://raw.githubusercontent.com/brokencube/ZipStream64/master/LICENCE
  *
  *
  * Requirements:
  *
- * * PHP version 5.1.2 or newer.
+ * PHP version 5.6 or newer
```

Comment:
We should remove this comment. The composer.json specifies that already.

Reply:
Removed.